automation/src/test/resources/diff-tests/hail-tutorial.ipynb

```diff
@@ -1,1644 +1,1006 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## GWAS Tutorial\n",
+    "\n",
+    "This notebook is designed to provide a broad overview of Hail's functionality, with emphasis on the functionality to manipulate and query a genetic dataset. We walk through a genome-wide SNP association test, and demonstrate the need to control for confounding caused by population stratification.\n"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "Running on Apache Spark version 2.0.2\n",
-      "SparkUI available at http://10.240.0.16:4040\n",
-      "Welcome to\n",
-      "     __  __     <>__\n",
-      "    / /_/ /__  __/ /\n",
-      "   / __  / _ `/ / /\n",
-      "  /_/ /_/\\_,_/_/_/   version 0.1-5c275cc\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "from hail import *\n",
-    "hc = HailContext(sc)"
+    "import hail as hl\n",
+    "import hail.expr.aggregators as agg\n",
+    "hl.init()"
```

Comment:
You may need to change this to `hl.init(sc)` so that it uses the already-running SparkContext.